admin_guide/garbage_collection.adoc

```diff
@@ -58,6 +58,20 @@ kubeletArguments:
 ----
 ====
 
+==== Detection of containers for deletion
+
+At the moment only docker and rkt are supported. Rkt has its own GC. The following is only docker related.
+
+First, garbage collector retrieves a list of available containers.
+Filters out all containers that are running or does not live longer then `minimum-container-ttl-duration` parameter.
+Then determines pod membership of each remaining container.
+Removes all unidentified containers (with no pod membership).
```

Comment:
same for these... maybe they should be put in a bulleted list:
The garbage collector
1) retrieves a list of available containers
2) filters out containers... 
3) ...